plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergParquetConnectorTest.java

```diff
@@ -73,4 +84,26 @@ protected Session withSmallRowGroups(Session session)
                 .setCatalogSessionProperty("iceberg", "parquet_writer_batch_size", "10")
                 .build();
     }
+
+    @Override
+    void verifyFileIsSorted(String path, String sortColumnName)
+    {
+        Comparable previousMax = null;
+        try (ParquetFileReader parquetReader = ParquetFileReader.open(HadoopInputFile.fromPath(new Path(path), ConfigurationInstantiator.newEmptyConfiguration()))) {
```

Comment:
Naive question: Why not read all the rows from the given parquet file and actually verify that any given row follows the sort order contract in respect to the previous row?

Reply:
Hmmm, we could it just sounds more complex to set up the test.